optuna/storages/_rdb/models.py

```diff
@@ -480,6 +480,27 @@ def where_trial_id(
         return trial_intermediate_values
 
 
+class TrialTimeStampModel(BaseModel):
+    __tablename__ = "trial_timestamp"
+    __table_args__: Any = (UniqueConstraint("trial_id", "step"),)
+    trial_timestamp_id = Column(Integer, primary_key=True)
+    trial_id = Column(Integer, ForeignKey("trials.trial_id"), nullable=False)
+    step = Column(Integer, nullable=False)
```

Comment:
Can we not stick to a single row per trial or is this step needed for the heartbeat?